hail_scripts/utils/clinvar.py

```diff
@@ -21,9 +40,68 @@
         "practice_guideline": 4,
     }
 )
+CLINVAR_PATHOGENICITIES_LOOKUP = hl.dict(
+    hl.enumerate(
+        # NB: sorted by pathogenicity
+        [
+            'Pathogenic',
+            'Pathogenic/Likely_pathogenic',
+            'Pathogenic/Likely_pathogenic/Likely_risk_allele',
+            'Pathogenic/Likely_risk_allele',
+            'Likely_pathogenic',
+            'Likely_pathogenic/Likely_risk_allele',
+            'Likely_risk_allele',
+            'Conflicting_interpretations_of_pathogenicity',
+            'Established_risk_allele',
```

Comment:
Wanted to call out that I added `Established_risk_allele` here, which isn't present in `CLNSIG` but is present in `CLNSIGCONF`.

Reply:
This should be sorted above `Likely_risk_allele`